gettsim/benefits/alg2.py

```diff
@@ -33,7 +31,7 @@ def alg2(household, tb):
     )
     # Aggregate on HH
     for var in ["ar_alg2_ek", "alg2_grossek", "uhv"]:
-        household[var + "_hh"] = aggr(household, var, "all_hh")
+        household[var + "_hh"] = household[var].sum()
```

Comment:
Go for ``f"{var}_hh"`` which is more readable.

Reply:
Converted to f-string